app/templates/teachers/teacher-student-view.jade

```diff
@@ -0,0 +1,220 @@
+extends /templates/base-flat
+
+block page_nav
+  include ../courses/teacher-dashboard-nav.jade
+
+block content
+
+  if !view.user
+    div Loading
+  else
+    .container
+      +breadcrumbs
+      if !me.isAnonymous() && me.isTeacher()
+      - var isOwner = view.classroom ? view.classroom.get('ownerID') === me.id : false;
+        // - var validStudent = ...  
+        // load all students in this class
+        // check if this student is in the class
+          
+      if isOwner
+        // also check if validStudent 
+        h3(class="m-t-2")
+          span(data-i18n="teacher.student_profile")
+          span.spr :
+          span= view.user.get('name')
+        // this user object doesn't have an email property
+        h4(class="student-details" data-i18n="teacher.student_overview")
+        if (view.user.get('email'))
+          div(class="small-details")
```

Comment:
While this is valid, in jade you generally do the shorthand that looks like a CSS selector: `div.small-details`. But even better, if you don't specify what kind of tag it is, it's `div` by default, so you can just do `.small-details`.